tests/Features/ReturnTypes/ContainerArrayAccessTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Features\ReturnTypes;
+
+use Illuminate\Auth\AuthManager;
+use Illuminate\Contracts\Foundation\Application;
+
+class ContainerArrayAccessTest
+{
+    /** @var Application */
+    protected $app;
+
+    public function testAuth(): ?AuthManager
```

Comment:
`auth` always exists on the container. I would prefer the test to be:

```
public function testAuth(): AuthManager
```

Reply:
It's because of [this](https://github.com/nunomaduro/larastan/blob/master/src/Properties/ReflectionTypeContainer.php#L43) line. I can change it to `false`. It doesn't seem to affect anything else. But not sure.

Reply:
If that lines specifies what can be resolved out of the container, a null can't be relsolved I guess.

Reply:
Changed.